### DISCUSSION TITLE
How can i Disable Kubernetes

### DISCUSSION THREAD

**Discussion by Drallas** [abiosoft/colima#597] [Q&A] [answered]
After I start Colima with the Kubernetes flag it keeps starting even I don't need it anymore. Even after deleting Kubernetes, it keeps on starting again! 

How can I prevent it from starting?

**Comment by abiosoft** [comment, marked as answer]
```
colima start --kubernetes=false
```

**Reply by ivanlawrence** [reply]
I also found it by running `colima start --edit` then finding the line about kubernetes and setting it back to it's default of `false`

**Comment by Drallas** [comment]
Where can I find documentation that matches this settings. 
Or is it yet undocumented?

**Reply by abiosoft** [reply]
I agree that documentation in general is a bit lacking. 

I am not sure if it is the standard unix way of disabling boolean flags but it is pretty much common for cli apps written in Go.
